cirq/experiments/qubit_characterizations.py

```diff
@@ -404,10 +402,10 @@ def _measurement(two_qubit_circuit: circuits.Circuit) -> np.ndarray:
         prob_list = [results_hist[0], results_hist[1], results_hist[2]]
         return np.asarray(prob_list) / repetitions
 
-    sigma_0 = np.eye(2) / 2.0
-    sigma_1 = np.array([[0.0, 1.0], [1.0, 0.0]]) / 2.0
-    sigma_2 = np.array([[0.0, -1.0j], [1.0j, 0.0]]) / 2.0
-    sigma_3 = np.array([[1.0, 0.0], [0.0, -1.0]]) / 2.0
+    sigma_0 = np.eye(2) * 0.5
```

Comment:
do these changes impact speed or anything?  Might move the scalar in front but that's a nit.

Reply:
They technically do but practically speaking you only get gains if the np array is a reasonably large size. I made the division of constant -> multiplication by inverse changes mostly out of force of habit.